src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -134,6 +135,9 @@ slots:
       optional field is documentation for human reviewers - when a category is known
       and documented clearly, the cost of interpreting and evaluating the mapping decreases.
     range: string
+    see_also:
+      - https://github.com/mapping-commons/sssom/issues/13
```

Comment:
While linking to related issues talking about fields as they were added, this is not what I was talking about. I want to see a link to a TSV file that demonstrates actual usage of the field in an actual SSSOM file.

Reply:
I added an example for this to the #258 pull request, with a link to an SSSOM file. We will make adding a properly formatted SSSOM file a requirement, but I don't think it has be be real. The discussion in the issue will have to make a case it is needed - the example should merely illustrate how it is intended to be used.